src/components/upgrader/ipUpgrader.ts

```diff
@@ -438,7 +438,7 @@ export default class IpCameraUpgrader extends EventEmitter implements IDeviceUpg
         Logger.warn(errMsg, this.className);
```

Comment:
While you are here, can you fix the errMsg to not hard code the seconds timeout (line 437)?

Reply:
Yup, good thinking :D